arch/mips/mips_features.c

```
@@ -0,0 +1,27 @@
+#if defined(MIPS_FEATURES)
+
+#include "zbuild.h"
+#include "mips_features.h"
+
+#ifdef HAVE_SYS_AUXV_H
+#  include <sys/auxv.h>
+#endif
+
+#include <asm/hwcap.h>
+
+void Z_INTERNAL mips_check_features(struct mips_cpu_features *features) {
```

Cppcheck (2.21.0)

[style] The function 'mips_check_features' is never used.

(unusedFunction)